### PR TITLE
[127] Made sure attributes ordering is preserved

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: framework/bundles/org.jboss.tools.rsp.server.spi/src/main/java/org/jboss/tools/rsp/server/spi/servertype/AbstractServerType.java [hunk 1]
```diff
@@ -13,17 +13,17 @@
...
 	private String name;
 	private String desc;
+
 	public AbstractServerType(String id, String name, String desc) {
 		this.id = id;
 		this.name = name;
 		this.desc = desc;
 	}
 
-	public abstract IServerDelegate createServerDelegate(IServer server);
```
Comment (robstryker) [inline framework/bundles/org.jboss.tools.rsp.server.spi/src/main/java/org/jboss/tools/rsp/server/spi/servertype/AbstractServerType.java]: Why remove?

File: framework/tests/org.jboss.tools.rsp.server.test/src/main/java/org/jboss/tools/rsp/server/model/ServerModelTest.java [hunk 2]
```diff
@@ -49,13 +50,14 @@
 public class ServerModelTest {
 	private static final DataLocationSysProp dataLocation = new DataLocationSysProp();
 
-	private static final String BONNET_COLOR_ATTRIBUTE_ID = "Bonnet";
+	private static final String BONNET_COLOR_ATTRIBUTE_ID = "BonnetColor";
 	private static final String HAS_BEARD_ATTRIBUTE_ID = "HasBeard";
+	protected static final String THROUSERS_COLOR_ATTRIBUTE_ID = "throusersColor";
```
Comment (robstryker) [inline framework/tests/org.jboss.tools.rsp.server.test/src/main/java/org/jboss/tools/rsp/server/model/ServerModelTest.java]: can you please tell me what "throusers" means? through the users?  I'm a bit confused.

File: framework/bundles/org.jboss.tools.rsp.server.spi/src/main/java/org/jboss/tools/rsp/server/spi/servertype/AbstractServerType.java (hunk 1 above)
Comment (adietish) [reply framework/bundles/org.jboss.tools.rsp.server.spi/src/main/java/org/jboss/tools/rsp/server/spi/servertype/AbstractServerType.java]: bcs this is already in the interface ```IServerType#createServerDelegate```. There's no need to repeat this in the abstract class.

**Comment by adietish** [comment]
@robstryker added the following: https://github.com/redhat-developer/rsp-server/pull/180/files#diff-6762acf320db92384aef97cff8c2d03bR218
please review again.

File: framework/tests/org.jboss.tools.rsp.server.test/src/main/java/org/jboss/tools/rsp/server/model/ServerModelTest.java (hunk 2 above)
Comment (adietish) [reply framework/tests/org.jboss.tools.rsp.server.test/src/main/java/org/jboss/tools/rsp/server/model/ServerModelTest.java]: "throusers" should be spelled "trousers" ;)

**Comment by adietish** [comment]
fixes #127

**Pull request merged**